src/content/configuration/stats.md

```diff
@@ -180,6 +180,27 @@ module.exports = {
 }
 ```
 
+If you want to use one of the pre-defined behaviours e.g. `'minimal'` but still override one or more of the rules, see [the source code](https://github.com/webpack/webpack/blob/master/lib/Stats.js#L1394-L1401). You would want to copy the configuration options from `case 'minimal': ...` and add your additional rules while providing an object to `stats`.
```

Comment:
I guess we could wrap **e.g. minimal** piece in parentheses `()`. What do you think?

Reply:
I know what you mean, but doesn't make much sense as the example is completely about `minimal`. The url, the case and webpack config